[expo-barcod-scanner] Return bounding box (#8865)
expo/expo: docs/pages/versions/unversioned/sdk/bar-code-scanner.md

#### Returns

A possibly empty array of objects of the shape `{ type: BarCodeScanner.Constants.BarCodeType, data: string }`, where the type refers to the bar code type that was scanned and the data is the information encoded in the bar code.

## Types

### `BarCodeScanner.BarCodePoint`

Object of type `BarCodePoint` contains following keys:

- **x (_number_)** -- The x value.
- **y (_number_)** -- The y value.

> Those coordinates are represented in the coordinate space of the barcode source. For example, when you are using the bar code scanner view, those values will be adjusted to this space.

Reviewer: ```suggestion
Those coordinates are represented in the coordinate space of the barcode source (e.g. when you are using the barcode scanner view, these values are adjusted to the dimensions of the view).
```
